### PR TITLE
Update related tags integration for compatibility with V3 index

### PR DESCRIPTION
**Pull request by i-like-robots**
https://github.com/Financial-Times/next-ft-api-client/pull/59 fixed this feature which had been failing since we deleted the V1 index. Well, here it is again...

screen shot 2016-02-09 at 17 32 03

### CODE REVIEW

File: server/controllers/v1-taxonomy.js [hunk 1]
```diff
@@ -217,7 +217,7 @@ export default coroutine(function *(req, res, next) {
 
 		// Decorate with related tags
 		if (stream.taxonomy && stream.tag.term.name) {
-			const tags = yield relatedTags(stream.taxonomy, stream.tag.term.name, req.params.thingId);
+			const tags = yield relatedTags(stream.tag.term.name);
```
Comment (andygnewman) [inline server/controllers/v1-taxonomy.js]: Is it valid to use name in this circumstance rather than id?

Comment (i-like-robots) [inline server/controllers/v1-taxonomy.js]: You're right, the label may be ambiguous. I'll need to update the client module first.

Comment (i-like-robots) [inline server/controllers/v1-taxonomy.js]: Sorted here: https://github.com/Financial-Times/next-ft-api-client/pull/65

**Comment by i-like-robots** [comment]
Published next-ft-api-client v3.21.1, just waiting for that to propagate.

**Comment by matthew-andrews** [comment]
really nicely done :+1:

**Pull request merged**